commit 03d0c794dc3a15e414c627f47a31718c39a49197
Author: Darrell Anderson <humanreadable@yahoo.com>
Date:   1332214935 -0500

    Fix configure output message to clarify that missing avahi support
    is caused by missing avahi-tqt package as well as avahi-client.

diff --git a/configure.in b/configure.in
index 29436e8..ca0b183 100644
--- a/configure.in
+++ b/configure.in
@@ -403,7 +403,8 @@ if test x$missing_avahi_warning = xyes; then
         echo ""
         echo "----------------------------------------------------------"
         echo "KTorrent WARNING:"
-        echo "Cannot find avahi-client with version 0.6.10 or later."
+        echo "Cannot find avahi-client with version 0.6.10 or later or"
+        echo "cannot find avahi-tqt."
 		echo "The zeroconf plugin will not be installed because of this."
         echo "----------------------------------------------------------"
         echo ""
diff --git a/configure.in.bot b/configure.in.bot
index b3f5186..15c2c8c 100644
--- a/configure.in.bot
+++ b/configure.in.bot
@@ -12,7 +12,8 @@ if test x$missing_avahi_warning = xyes; then
         echo ""
         echo "----------------------------------------------------------"
         echo "KTorrent WARNING:"
-        echo "Cannot find avahi-client with version 0.6.10 or later."
+        echo "Cannot find avahi-client with version 0.6.10 or later or"
+        echo "cannot find avahi-tqt."
 		echo "The zeroconf plugin will not be installed because of this."
         echo "----------------------------------------------------------"
         echo ""
